tests/extensions/python_test_files/multi_line.pyt

```diff
@@ -0,0 +1,15 @@
+"""
+Docstring
+
+.. pii: A long description that
+  spans multiple
+  lines
+.. pii_types: id, name
+"""
+
+"""
+    .. pii: A long description that
+      spans multiple indented
+      lines
+.. pii_types: id, name
```

Comment:
Is this legal?  What happened before with?  Seems like a weird example.:
```
    .. pii: A short description
.. pii_types: id, name
```

Reply:
Indeed, this is a little weird. I'll also indent the `.. pii_types` line to make it more coherent.